src/V3AstNodes.cpp

```diff
@@ -791,6 +808,9 @@ AstNodeDType::CTypeRecursed AstNodeDType::cTypeRecurse(bool compound) const {
         } else if (dtypep->isWide()) {
             info.m_type = "VlWide<" + cvtToStr(dtypep->widthWords()) + ">" + bitvec;
         }
+        // CData, SData, IData, QData or VlWide are packed type.
+        bool packedType = info.m_type[2] == 'a' || info.m_type[2] == 'W';
```

Comment:
Although slower I'd prefer an e.g. 4-character prefix match to the various possibilities as this seems otherwise sensitive to possible future type string changes.

Reply:
Fixed with more robust check
```c++
        bool packedType = VString::startsWith(info.m_type, "CData")
                          || VString::startsWith(info.m_type, "SData")
                          || VString::startsWith(info.m_type, "IData")
                          || VString::startsWith(info.m_type, "QData")
                          || VString::startsWith(info.m_type, "VlWide");
```